code/MathSpamProtectorField.php

```diff
@@ -101,17 +107,20 @@ public function validate($validator)
      *
      * @return string
      */
-    public static function get_math_question()
+    public function get_math_question()
```

Comment:
Could you please rename this method to `getMathQuestion()` (PSR-2 compliant) while you're removing its static signature?

Reply:
Oh, true - I didn't run a PSR2 sweeper